apps/src/templates/sectionProgressV2/ItemType.jsx

```diff
@@ -7,7 +7,7 @@ export const ITEM_TYPE = Object.freeze({
   NEEDS_FEEDBACK: 3,
   FEEDBACK_GIVEN: 4,
   ASSESSMENT_LEVEL: ['star', color.neutral_dark],
-  CHOICE_LEVEL: ['split', color.neutral_dark60],
+  CHOICE_LEVEL: ['split', color.neutral_dark60, color.neutral_dark],
```

Comment:
This felt like the least invasive way to change this color.  It does mean it doesn't match the pattern of other icons here, but I could see a world where we rely on more colors in the future too...

Reply:
I suggest a `colorOverride` parameter instead. We'll keep the defaults here, and anywhere that we need to use a different color we can specify that when rendering the `ProgressIcon` component.